apps/explorer/src/pages/object-result/ObjectResult.tsx

```diff
@@ -93,7 +94,11 @@ function ObjectResultAPI({ objID }: { objID: string }) {
     }, [objID, rpc]);
 
     if (showObjectState.loadState === 'loaded') {
-        return <ObjectView data={showObjectState as DataType} />;
+        return (
+            <ErrorBoundary>
```

Comment:
I feel like this boundary more logically exists in the parent

Reply:
done